### DISCUSSION TITLE
Learning Online-3D-BPP-DRL - Get amount of the used containers

### DISCUSSION THREAD

**Discussion by alex-ams** [community/community#30804] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, 

I'm studying the 3D Bin packing with this repository: [https://github.com/alexfrom0815/Online-3D-BPP-DRL].
I have some troubles with the evaluation.py file. I'm using my own data by generating a ".pt" but I don't know how to retrieve the number of used containers and the number of items packed in each one. 
Could you please help with this? 

Thanks in advance

**Comment by jfullstackdev** [comment, marked as answer]
maybe you can raise an issue there to ask that question too

https://github.com/alexfrom0815/Online-3D-BPP-DRL/issues
